packages/next/taskfile.js

```diff
@@ -2245,22 +2245,24 @@ export async function precompile(task, opts) {
     opts
   )
 
-  const validatorRes = await fetch(
-    'https://cdn.ampproject.org/v0/validator_wasm.js'
-  ).catch((err) => {
-    throw new Error('Failed to fetch AMP validator', { cause: err })
-  })
```

Comment:
I feel like the better fix would be to use a permalink and check this file into version control. Otherwise it's always a reproducibility concern.

Reply:
Agreed. I've updated the PR (including its description) accordingly.